src/renderer/families/stellar/MemoValueField.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+import React, { useCallback } from "react";
+import { getAccountBridge } from "@ledgerhq/live-common/lib/bridge";
+import Input from "~/renderer/components/Input";
+import invariant from "invariant";
+import type { Account, Transaction, TransactionStatus } from "@ledgerhq/live-common/lib/types";
+
+const MemoValueField = ({
+  onChange,
+  account,
+  transaction,
+  status,
+}: {
+  onChange: string => void,
+  account: Account,
+  transaction: Transaction,
+  status: TransactionStatus,
+}) => {
+  invariant(transaction.family === "stellar", "MemoTypeField: stellar family expected");
+
+  const bridge = getAccountBridge(account);
+
+  const onMemoValueChange = useCallback(
+    memoValue => {
+      onChange(bridge.updateTransaction(transaction, { memoValue }));
+    },
+    [onChange, transaction, bridge],
+  );
+
+  // We use transaction as an error here.
+  // It will be usefull to block a memo wrong format
+  // on the ledger-live mobile
+  const { transaction: memoError } = status.errors;
+  const { transaction: memoWarning } = status.warnings;
+
+  return (
+    <Input
+      warning={memoWarning}
+      error={memoError}
```

Comment:
totally minor here but to make it clearer i would directly inline the stuff (i'm not usually fan of rewriting the var name in spread, for readability), like

```
<Input
  warning={status.warnings.transaction}
  error={status.errors.transaction}
```